manifests/b/Bitnami/Drupal/9.4.3-0/Bitnami.Drupal.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Bitnami.Drupal
+PackageVersion: 9.4.3-0
+PackageLocale: en-US
+Publisher: Bitnami
+PublisherUrl: https://bitnami.com/
+PublisherSupportUrl: https://bitnami.com/support
+PrivacyUrl: https://www.vmware.com/help/privacy.html
+# Author: 
+PackageName: Bitnami Drupal Stack
```

```suggestion
PackageName: Drupal packaged by Bitnami
```
This must match control panel